polkadot/xcm/xcm-builder/src/fungible_adapter.rs

```diff
@@ -96,11 +96,13 @@ impl<
 
 	fn accrue_checked(checking_account: AccountId, amount: Fungible::Balance) {
 		let ok = Fungible::mint_into(&checking_account, amount).is_ok();
+		<Fungible as fungible::Unbalanced<_>>::deactivate(amount);
```

Comment:
In case we go with this fix, we should probably consider a migration/correction for `InactiveIssuance`, right? - see the comment: https://github.com/paritytech/polkadot-sdk/issues/8055#issuecomment-2756040541

Reply:
How about introducing a decorator type - similar to `accrue_checked` or an implementation of `TransactAsset` - that wraps types with `deactivate/reactivate` logic? This would make support for this feature more explicit. These types could be used for purposes beyond just teleports.

Reply:
we have two high rank Fellows offering to not deactivate teleported balance. I have no arguments against. https://github.com/paritytech/polkadot-sdk/issues/8055

if we go with this decision, we will need to remove those deactivate from `CurrencyAdapter` and  have some migration to correct the current value.

Reply:
@franciscoaguirre please add this PR to your TODO list to bring it to some final conclusion